### DISCUSSION TITLE
Transfer Subdomain

### DISCUSSION THREAD

**Discussion by VaibhavSys** [is-a-dev/register#3009] [General]
@Vaibhav-Kashyap Hey, I think that you are not using https://vaibhav.is-a.dev and I would like to use that subdomain, could I please have it?

**Comment by omsenjalia** [comment]
> @Vaibhav-Kashyap Hey, I think that you are not using https://vaibhav.is-a.dev and I would like to use that subdomain, could I please have it?

It says it's under development so they are probably doing something with it. Why don't you use mouse.is-a.dev

**Reply by VaibhavSys** [reply]
> Why don't you use mouse.is-a.dev

I have changed my usernames
